packages/connect/src/api/ethereum/api/ethereumCall.ts

```diff
@@ -0,0 +1,65 @@
+// origin: https://github.com/trezor/connect/blob/develop/src/js/core/methods/EthereumCall.js
+
+import { AbstractMethod } from '../../../core/AbstractMethod';
+import { validateParams } from '../../common/paramsValidator';
+import { ERRORS } from '../../../constants';
+import { CoinInfo, EthereumCall as EthereumCallSchema } from '../../../types';
+import { initBlockchain, isBackendSupported } from '../../../backend/BlockchainLink';
+import { getCoinInfo } from '../../../data/coinInfo';
+
+type Params = {
+    coinInfo: CoinInfo;
+    identity?: string;
+    request: EthereumCallSchema;
+};
+
+export default class EthereumCall extends AbstractMethod<'ethereumCall', Params> {
```

Comment:
why not prefix it with `blockchain..` similarly to other blockchain-link interfacing methods?